pymlx/featurizer.py

```diff
@@ -201,8 +205,10 @@ def suggest_handlers(df, sample_size=10000, trees_optimized=True, hinted_featuri
         handlers[kind].append(col)
 
     ret = AttrDict()
+    # NOTE THIS
```

Comment:
Note what?

Reply:
note that the code `import pandas as pd` but use `pandas._` :/ this causes an error.
```
pandas.options.display.float_format = '{:.2g}'.format
pandas.options.display.max_rows = 500
```
that why I have used `import pandas` to debug the code